app/models/work.rb

```diff
@@ -0,0 +1,28 @@
+class Work < ApplicationRecord
+  has_many :votes, dependent: :destroy
+  # has_many :users, through :votes
+
+  validates :title, presence: true, uniqueness: {scope: :category}
+  validates :publication_year, numericality: true, length: {is: 4}
+  validates :creator, presence: true
+
+  def self.sort_by_category(category)
```

Comment:
The name `sort_by_category` doesn't sound right since this method doesn't really do sorting.  Instead it does filtering.